prometheus/prometheus.ini

```diff
@@ -1,6 +1,6 @@
 [manifest]
 potname="prometheus"
 author = "Stephan Lichtenauer, Bretton Vine, Michael Gmelin"
-version="0.15.1"
+version="0.16.0"
```

Comment:
I'm incrementing X for new base image changes and Y for image updates, so 0.15.2 perhaps?

0.X.Y

Reply:
Ok, I updated it to match your convention.